services/init-protonet.service

```diff
@@ -27,7 +27,7 @@ ExecStartPre=/bin/mkdir -p /data/soul/public
 ExecStartPre=/bin/mkdir -p /data/soul/uploads
 ExecStartPre=/bin/mkdir -p /data/soul/protosync
 ExecStartPre=/bin/mkdir -p /data/pgp
-ExecStartPre=/bin/bash -c '[[ ! -d /backup ]] && zfs create -p -o mountpoint=/backup protonet_storage/backup'
+ExecStartPre=-/bin/bash -c '[[ ! -d /backup ]] && zfs create -p -o mountpoint=/backup protonet_storage/backup'
```

Comment:
I had this one failing on the AMI tho. that's why I changed it. I'll double check if I mixed something up here